test/tasks/Regression.t.sol

```diff
@@ -110,6 +111,24 @@ contract RegressionTest is Test {
         assertEq(keccak256(bytes(dataToSign)), keccak256(bytes(expectedDataToSign)));
     }
 
+    function testRegressionCallDataMatches_EnableDeputyPauseModuleTemplate() public {
```

Comment:
Is the developer UX that part of template creation requires adding a corresponding regression test? That seems reasonable to me if so, but just want to confirm

Reply:
Sanity checking we are working on "template creation docs" that explain this? It also looks like these tests duplicate tons of code, we probably can make it a lot easier to implement and maintain these tests with some helper functions

Reply:
The commit above doesn't include a refactor of `Regression.t.sol` with helper functions to avoid code duplication. I've raised an issue for this to be tackled later: https://github.com/ethereum-optimism/superchain-ops/issues/655

Reply:
Resolving.